src/java.base/share/classes/java/util/function/BiFunction.java

```diff
@@ -24,6 +24,7 @@
  */
 package java.util.function;
 
+import java.lang.FunctionalInterface;
```

Comment:
@mernst @wmdietl 
This is one additional problem I am able to discover
AUF is adding extra `@FunctionaInterface` to every CLASS of `java/util/function` which was already present.
The JAIF files contained these annotations.
Any idea how to remove these?

Thanks.

Reply:
To make my question more precise I want `extract-annotation <annotatedJDK8>` to give JAIF that only contains annotations from checker-framework like `org.checkerframework...`

While using `insert-annotations-to-source` I used `--in-place=true` option.

Thanks.